### PR TITLE
Add seed to runit_mean_axis.R 

### PR DESCRIPTION
**Pull request by navdeep-G**
* This PR adds a proper seed to `runit_mean_axis.R`. Should help runs in CI
* Also, this PR adds print statements where relevant

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/539)

### CODE REVIEW

**Comment by navdeep-G** [comment]
@h2o-ops please test!

**Comment by navdeep-G** [comment]
@h2o-ops please test!

**Comment by navdeep-G** [comment]
@h2o-ops please test!

**Comment by navdeep-G** [comment]
@h2o-ops please test!

File: h2o-r/tests/testdir_munging/unop/runit_mean_axis.R [hunk 1]
```diff
@@ -10,7 +10,7 @@ test.mean <- function(){
   fr <- h2o.createFrame(rows = 100, cols = 10, categorical_fraction = 0,
                     factors = 0, integer_fraction = 1.0, integer_range = 100,
                     binary_fraction = 0, time_fraction = 0, string_fraction = 0,
-                    has_response = FALSE)
+                    has_response = FALSE,seed = 123456789)
```
Comment (mmalohlava) [inline h2o-r/tests/testdir_munging/unop/runit_mean_axis.R]: Just for tracking: the static seed hides the failure.

**Pull request merged**